### PR TITLE
Create method in AbstractController for resolve multiple template 

### LINKED ISSUE #9071: Allow using multiple template candidates when rendering twig template (like include)
```php
// there is no possibility do smth like this
$templates = array(
    sprintf('MissProjectBundle:StaticPage:custom/%s', $page['template']),
    'MissProjectBundle:StaticPage:static_page_layout.html.twig'
);

return $this->render($templates);

// and we should use workaround
return new Response($this->get('twig')->resolveTemplate($templates)->render(array()));
```

### PR DESCRIPTION
**Pull request by peter-gribanov**
| Q             | A
| ------------- | ---
| Branch?       | 5.0
| Bug fix?      | no
| New feature?  | yes 
| Deprecations? | no
| Tickets       | Fix #9071
| License       | MIT
| Doc PR        | 

Usage like this:

```php
return $this->resolveRender([
    sprintf('@Frontend/Broadcast/skin/%s/show.html.twig', $broadcast->getLayout()),
    '@Frontend/Broadcast/skin/default/show.html.twig',
]);
```

While this is a feature for discussion.

### CODE REVIEW

**Comment by yceruto** [comment]
Hi @peter-gribanov, new features should be sent to the `master` branch, thanks.

**Comment by peter-gribanov** [comment]
@yceruto thanks. I change base branch.

File: src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php [hunk 1]
```diff
@@ -264,6 +264,38 @@ protected function render(string $view, array $parameters = [], Response $respon
         return $response;
     }
 
+    /**
+     * Returns a rendered view resolved for several templates.
+     */
+    protected function resolveRenderView(array $views, array $parameters = []): Response
```
Comment (andrew-demb) [inline src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php]: Isn't this should return `: string` ?

Comment (peter-gribanov) [reply src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php]: Thanks. I made a typo

**Comment by nicolas-grekas** [comment]
I'm sorry but this doesn't look common enough to be added to the base controller. We'd prefer keeping it as small as possible.

**Comment by peter-gribanov** [comment]
@nicolas-grekas I agree. I also see this as a highly specialized feature. But i would not want this code to get lost. I needed this feature for many years and only yesterday i accidentally discovered that i could do this.

**Comment by fabpot** [comment]
i agree with @nicolas-grekas

**Comment by a-menshchikov** [comment]
Maybe this code could be placed into trait?

@nicolas-grekas what is your opinion?

**Pull request closed without merging**